### DISCUSSION TITLE
Dangerous Fake Repo found.

### DISCUSSION THREAD

**Discussion by CatWithCode** [rocksdanister/lively#1595] [General]
There is an GitHub Repo by another user that seems to be a virus.

It is the first thing showing up when searching for lively wallpaper on DuckDuckGo and Google sometimes.

(The "-" are for safety so no ones exadently clicks it and downloads the wrong thing by accident).
h-t-t-p-s:-/-/-g-i-t-h-u-b-.-c-o-m-/-L-i-v-e-l-y-W-a-l-l-p-a-p-e-r-/-l-i-v-e-l-y-

There seems to be a chain of scripts downloading a virus:
h-t-t-p-s-:-/-/-g-i-t-h-u-b-.-c-o-m-/-L-i-v-e-l-y-W-a-l-l-p-a-p-e-r-/-l-i-v-e-l-y-/-i-s-s-u-e-s-/-4

If I'm wrong or misinterpreted something please just delete this.

**Comment by rocksdanister** [comment]
I am aware (although I don't know if its malware or not.. the script file is downloading and executing code from a url so it is suspicious.) 
You can try reporting the profile (using any github name like LivelyWallpaper is allowed but you can't distribute malware.)

Looking at the numbers almost 500 people downloaded from there:
https://tooomm.github.io/github-release-stats/?username=livelywallpaper&repository=lively

**Comment by rocksdanister** [comment]
Not sure what happened but it looks like the user account and repo is deleted https://github.com/LivelyWallpaper

<img width="733" alt="image" src="https://user-images.githubusercontent.com/17554161/222454492-7ba30dc3-d5ca-4e5b-b9d0-e9c729ac6292.png">

**Reply by CatWithCode** [reply]
Seems like enough people reported it.

**Comment by rocksdanister** [comment]
New one is up:
https://github.com/Lively-wallpaper/lively
